### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/canonical-raft/commit/90fad1b6e2f550dac107bc50d12e8b305404a761#commitcomment-43339043

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/canonical-raft/commit/c8fba5e8ad2b78c51b385ee7c128599bc4643ad3

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: test/lib/munit.c [hunk 1]
```diff
@@ -308,7 +308,7 @@ munit_malloc_ex(const char* filename, int line, size_t size) {
  * sync. */
 
 /* Clocks (v1)
- * Portable Snippets - https://gitub.com/nemequ/portable-snippets
```
Comment (jsoref) [inline test/lib/munit.c:311]: This domain has a squater

File: src/uv_fs.c [hunk 2]
```diff
@@ -529,7 +529,7 @@ static int probeDirectIO(int fd, size_t *size, char *errmsg)
         switch (fs_info.f_type) {
             case 0x01021994: /* TMPFS_MAGIC */
             case 0x2fc12fc1: /* ZFS magic */
-            case 0x24051905: /* UBIFS Suprt magic */
+            case 0x24051905: /* UBIFS Support magic */
```
Comment (jsoref) [inline src/uv_fs.c:532]: It's possible this is wrong, but ...
I also wonder if the word is necessary at all.

File: test/integration/test_recover.c [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 /******************************************************************************
  *
- * Fixture holding a bootstrapd raft caluster.
+ * Fixture holding a bootstrapped raft cluster.
```
Comment (jsoref) [inline test/integration/test_recover.c:6]: I'm not entirely certain about `bootstrapped`, it could be `bootstrap`...

File: test/lib/munit.c [hunk 4]
```diff
@@ -1672,7 +1672,7 @@ munit_test_runner_run_test(MunitTestRunner* runner,
 }
 
 /* Recurse through the suite and run all the tests.  If a list of
- * tests to run was provied on the command line, run only those
+ * tests to run was provide on the command line, run only those
```
Comment (jsoref) [inline test/lib/munit.c:1675]: Sigh, this should be `provided`

**Review by jsoref — COMMENTED** [review]
Most corrections were made by Google Sheets. It isn't perfect, but tbh my personal error rate is worse than its. Responsibility for incorrect corrections is of course mine.

File: src/uv.c [hunk 5]
```diff
@@ -46,7 +46,7 @@ static int uvInit(struct raft_io *io, raft_id id, const char *address)
         return rv;
     }
 
-    /* Proble file system capabilities */
+    /* Problem file system capabilities */
```
Comment (bl-ue) [inline src/uv.c:49]: @jsoref this looks likes it was intended to be `probe` (`UvFsProbeCapabilities`)

Comment (jsoref) [reply src/uv.c]: Thanks @bl-ue

File: src/uv_fs.c (hunk 2 above)
Comment (stgraber) [reply src/uv_fs.c:532]: not strictly needed indeed but that's fine

File: test/integration/test_recover.c (hunk 3 above)
Comment (stgraber) [reply test/integration/test_recover.c:6]: bootstrapped is fine

**Comment by stgraber** [comment]
Looks good to me, merging, thanks!

**Pull request merged**